<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_PluginAttrComment</class>
 <widget class="QDialog" name="W_PluginAttrComment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>属性</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget_title{ background-color: rgb(255, 255, 255); }
#widget_title_2{ background-color: rgb(255, 255, 255); }
#widget_title_3{ background-color: rgb(255, 255, 255); }
#widget_title_4{ background-color: rgb(255, 255, 255); }
#widget_title_5{ background-color: rgb(255, 255, 255); }
#widget_title_6{ background-color: rgb(255, 255, 255); }
#widget_title_7{ background-color: rgb(255, 255, 255); }
#widget_title_8{ background-color: rgb(255, 255, 255); }
#widget_title_9{ background-color: rgb(255, 255, 255); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_title" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_icon" native="true">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/DrillPluginInformationReader/Resources/icons/menu/attr_word.png);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_title">
            <property name="styleSheet">
             <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:18px;
font-weight:bold;
padding:4px;</string>
            </property>
            <property name="text">
             <string>插件文档</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_context" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>该插件有专门的文档支持。</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_docs"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_title_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_icon_2" native="true">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/DrillPluginInformationReader/Resources/icons/menu/attr_src.png);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_title_2">
            <property name="styleSheet">
             <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:18px;
font-weight:bold;
padding:4px;</string>
            </property>
            <property name="text">
             <string>资源路径</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_context_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_src"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_title_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_icon_3" native="true">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/DrillPluginInformationReader/Resources/icons/menu/attr_highConsumption.png);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_title_3">
            <property name="styleSheet">
             <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:18px;
font-weight:bold;
padding:4px;</string>
            </property>
            <property name="text">
             <string>高消耗</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_context_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>该插件性能消耗较高，使用时注意你的需求。</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_title_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_icon_4" native="true">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/DrillPluginInformationReader/Resources/icons/menu/attr_hasTool.png);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_title_4">
            <property name="styleSheet">
             <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:18px;
font-weight:bold;
padding:4px;</string>
            </property>
            <property name="text">
             <string>小工具支持</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_context_4" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>该插件可以支持额外的小工具编辑功能。</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_title_5" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_icon_5" native="true">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/DrillPluginInformationReader/Resources/icons/menu/attr_foreignKey.png);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_title_5">
            <property name="styleSheet">
             <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:18px;
font-weight:bold;
padding:4px;</string>
            </property>
            <property name="text">
             <string>外键</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_context_5" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>该插件配置中，含有其它插件配置的关联。
比如 xx样式、xx弹道、xx框架 配置。</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../drillplugininformationreader.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>W_PluginAttrComment</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
